packages/mux-player/src/media-theme-mux/styles.css

```diff
@@ -17,14 +26,16 @@ media-pip-button[media-pip-unavailable] {
 }
 
 media-controller {
-  --media-control-background: transparent;
-  --media-control-hover-background: transparent;
-  --media-range-track-background-color: rgba(255, 255, 255, 0.5);
-  --media-range-track-border-radius: 3px;
   width: 100%;
   height: 100%;
 }
 
+media-time-range {
+  z-index: 10;
```

Comment:
nit(non-blocking) if this is just for the hover preview, could we apply the z-index only to it?